ertviz/controllers/multi_parameter_controller.py

```diff
@@ -1,37 +1,25 @@
 import dash
 from dash.exceptions import PreventUpdate
-from dash.dependencies import Input, Output, State
+from dash.dependencies import Input, Output, State, MATCH
+import plotly.graph_objects as go
 from ertviz.models import EnsembleModel, MultiHistogramPlotModel, load_ensemble
-
-
-def _prev_value(current_value, options):
-    try:
-        index = options.index(current_value)
-    except ValueError:
-        index = None
-    if index > 0:
-        return options[index - 1]
-    return current_value
-
-
-def next_value(current_value, options):
-    try:
-        index = options.index(current_value)
-    except ValueError:
-        index = None
-    if index < len(options) - 1:
-        return options[index + 1]
-    return current_value
+import ertviz.assets as assets
 
 
 def multi_parameter_controller(parent, app):
     @app.callback(
-        Output(parent.uuid("parameter-selector"), "options"),
+        [
+            Output(parent.uuid("parameter-selector"), "options"),
+            Output(parent.uuid("parameter-selector"), "value"),
+        ],
         [
             Input(parent.uuid("ensemble-selection-store"), "data"),
+            Input(parent.uuid("response-observations-check"), "value"),
         ],
+        [State(parent.uuid("plot-selection-store"), "data")],
     )
-    def update_parameter_options(selected_ensembles):
+    def update_parameter_options(selected_ensembles, _, prev_plot_selection):
```

Comment:
Do we want an underscore prefix here? (And on the rest of all callback functions?)

Reply:
I would actually rather have it only on functions in the files that are _not_ callbacks ;)

Reply:
I think that makes sense :+1:  I was just wondering as some callback functions has underscore prefixes and some does not.

Reply:
I made a commit to clean up the style :)

Reply:
Nice! I like consistency :+1: